resource/history/mage-os/product-community-edition/1.1.0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "require": {
+    "mage-os/security-package": "1.1.0",
+    "mage-os/inventory-metapackage": "1.1.0",
+    "mage-os/page-builder": "1.1.0"
```

Comment:
I think this needs the admin theme and interceptors modules added per https://github.com/mage-os/generate-mirror-repo-js/blob/main/resource/composer-templates/mage-os/product-community-edition/dependencies-template.json

Reply:
I want to document that cause I wasn't aware

Reply:
@rhoerr is the new version ok?